action.yml

```diff
@@ -7,7 +7,8 @@ inputs:
   environment-file:
     description: >-
       The environment.yml or .lock file for the conda environment.
-      If 'false', no enviroment will be created (only Micromamba will be installed).
+      If 'false', no enviroment will be created (only Micromamba will be installed)
```

Comment:
I think this might be too late, but could we also use `null`?

Reply:
It seems that GHA deletes inputs with `foo: null` from the actual inputs that are being passed to the action. `foo: 'null'` would work but feels ugly. So I suggest to keep `false`.